diff --git a/configure b/configure
index 7476ca8..856cacc 100755
--- a/configure
+++ b/configure
@@ -3578,8 +3578,8 @@ case "$toolchain" in
         add_cppflags -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2
         add_cflags   -fno-strict-overflow -fstack-protector-all
         add_ldflags  -Wl,-z,relro -Wl,-z,now
-        add_cflags   -fPIE
-        add_ldexeflags -fPIE -pie
+#        add_cflags   -fPIE
+#        add_ldexeflags -fPIE -pie
     ;;
     ?*)
         die "Unknown toolchain $toolchain"
